tools/upgrade-version/src/main.rs

```diff
@@ -145,21 +50,29 @@ fn main() {
     let current_dir = env::current_dir().expect("No current directory!");
     let dir_name = current_dir.file_name().expect("No current directory!");
     if dir_name != "SpacetimeDB" {
```

Comment:
Before merging: please change this to `dir_name != "SpacetimeDB" && dir_name != "public"`